rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -959,6 +936,43 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		propagate_used_devices();
 	};
 
+	const auto get_audio_output_devices = [this]()
+	{
+		auto dev_enum = Emu.GetCallbacks().get_audio_enumerator();
```

Comment:
This is not working as intended.
The callback should take the enum value as param.
The global config is obviously not updated yet when you change the audio backend.